scripts/circleci/get_tagged_package.py

```diff
@@ -0,0 +1,64 @@
+# Copyright 2016 Google LLC
```

Comment:
`scripts/` can safely be removed. These were all made obsolete by repo split

Reply:
Removed!